src/lib/discord/commando/commands/autocreate.js

```diff
@@ -75,8 +75,13 @@ exports.run = async (client, msg, [args]) => {
 
 		let categoryId
 		if (template.definition.category) {
-			const category = await guild.channels.create(format(template.definition.category, args), { type: 'GUILD_CATEGORY' })
-			categoryId = category.id
+			const exists = await guild.channels.cache.find(channel => channel.type === 'GUILD_CATEGORY' && channel.name === template.definition.category)
```

Comment:
somehow this does not give any result, because it never finds existing categories, it always creates a new one with identical name

Reply:
The original code is intended to create new categories.  You will notice you are comparing to the raw string name not an expanded one as well

Reply:
Yeah it depends on the template used. I used a template with a variable category, I define it as param :)

That's why I thought about fetching existing categories first, but I will remove that part.... The deny is fixed and worked as intended :)